src/extensions/extension-manager.ts

```diff
@@ -8,9 +8,10 @@ import { extensionPackagesRoot } from "./extension-loader"
 import { getBundledExtensions } from "../common/utils/app-version"
 
 export interface InstalledExtension {
-  manifest: LensExtensionManifest;
-  manifestPath: string;
-  isBundled?: boolean; // defined in package.json
+  readonly manifest: LensExtensionManifest;
+  readonly manifestPath: string;
+  readonly isBundled?: boolean; // defined in project root's package.json
```

Comment:
nit: still don't think this should be optional

Reply:
yep, forgot to remove `?`